iwyu_globals.cc

```diff
@@ -405,6 +410,21 @@ bool ShouldReportIWYUViolationsFor(const clang::FileEntry* file) {
   return false;
 }
 
+void AddGlobToKeepIncludes(const string& glob) {
+  CHECK_(commandline_flags && "Call ParseIwyuCommandlineFlags() before this");
+  commandline_flags->keep.insert(NormalizeFilePath(glob));
+}
+
+bool ShouldKeepIncludeFor(const clang::FileEntry* file) {
+  if (GlobalFlags().keep.size()) {
```

Comment:
I'd prefer to early exit here, and use `empty()` instead of `size()`:

    if (GlobalFlags().keep.empty())
      return false;